src/components/Header/Header.js

```diff
@@ -1,17 +1,39 @@
 import React from 'react'
 import HeaderNavigation from '../HeaderNavigation'
 import logo from '../../assets/images/logo.png'
+import {useNavigate} from 'react-router-dom';
+import {scrollTop} from '../../helpers';
 import cx from 'classnames'
 import st from './Header.module.scss'
-import Button from '../Button';
 
-const MENU = ['Menu 1', 'Menu 2', 'Menu 3', 'Menu 4'];
+const MENU = [
+  {
+    title: 'Main',
+    path: '/'
+  },
+  {
+    title: 'Characters',
+    path: 'characters'
+  },
+  {
+    title: 'About',
+    path: 'about',
+  },
+  {
+    title: 'Contacts',
+    path: 'contacts'
+  },
+
+  ];
+
 const Header = () => {
+  const navigate = useNavigate()
   const [scroll, setScroll] = React.useState(false)
 
   const onScroll = () => {
     window.scrollY > 60 ? setScroll(true) : setScroll(false)
```

Comment:
Можно написать короче :)
```suggestion
    setScroll(window.scrollY > 60)
```